app/views/reports/regions/index.html.erb

```diff
@@ -15,10 +15,26 @@
       <section>
         <div class="card">
           <h2><%= organization.name %></h2>
-          <% policy_scope([:cohort_report, organization.facility_groups]).order(:name).each do |district| %>
-            <%= link_to(reports_region_path(district, report_scope: "district"), class: "link-row") do %>
-              <i class='fas fa-angle-right light'></i>
-              <%= district.name %>
+          <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
+            <% current_admin.accessible_facilities(:view_reports)
+               .includes(facility_group: :organization)
+               .where(facility_groups: {organization: organization})
+               .flat_map(&:facility_group)
+               .uniq
+               .compact
```

Comment:
Same here, if we could call `accessible_facility_groups` that would be nice.

Reply:
_ditto_ ^